packages/suite/src/reducers/wallet/graphReducer.ts

```diff
@@ -79,6 +80,11 @@ const remove = (draft: State, accounts: Account[]) => {
 
 const graphReducer = (state: State = initialState, action: WalletAction | SuiteAction): State =>
     produce(state, draft => {
+        if (accountsActions.removeAccount.match(action)) {
+            remove(draft, action.payload);
+            return;
```

Comment:
In case of reducers I would prefer not to break `switch` convention, please place this back into switch and you should `if` also as type guard